.github/workflows/build_and_test.yaml

```diff
@@ -109,6 +109,14 @@ jobs:
         kubectl cp sidecar-5xx:/tmp-5xx/relative/relative.txt /tmp/5xx/relative.txt
         kubectl cp sidecar-5xx:/tmp-5xx/500.txt /tmp/5xx/500.txt
 
+    - name: Update Configmaps and Secrets
+      run: |
+        # because the sidecar pods signal ready state before we actually opened up all watching subprocesses, we wait some more time
```

Comment:
please remove this comment as it does not apply here. I believe that the sleep invocation is obsolete, too.

Reply:
updated